docs/src/SBP_Operators.md

```diff
@@ -0,0 +1,182 @@
+
+Several classes of numerical methods can be formulated via SBP operators, including finite difference methods, finite volume methods, continuous Galerkin methods, discontinuous Galerkin (DG) methods, and flux reconstruction methods. This unifying framework has made SBP operators a cornerstone of structure-preserving numerical methods across various computational disciplines.
+
+### Why SBP Operators Matter for Dispersive Shallow Water Equations
+
+For dispersive shallow water equations implemented in DispersiveShallowWater.jl, maintaining physical properties like mass and energy conservation is crucial for:
+
+- **Long-time stability**: Simulations remain stable over extended time periods without spurious growth of numerical errors
+- **Physical accuracy**: The discrete solution respects fundamental physical laws such as conservation of mass, momentum, and energy
+- **Robustness**: Methods are less sensitive to parameter choices and grid resolution, leading to more reliable simulations
+
+SBP operators achieve this by providing:
+
+- **Provable stability**: Mathematical guarantees about the behavior of the numerical method through discrete energy estimates
+- **Exact conservation**: Discrete conservation laws that hold to machine precision
+- **Flexibility**: A unified framework that encompasses finite differences, finite elements, and spectral methods
+
+### Historical Context and Development
+
+The development of SBP operators represents a significant advancement in bridging the gap between the theoretical foundations of finite element methods and the computational efficiency of finite difference methods. While finite element methods naturally possess stability properties through their variational formulation, finite difference methods traditionally lacked such systematic guarantees. SBP operators provide the missing link by bringing the stability analysis tools of finite element methods to finite difference and other numerical approaches.
+
+The framework has since been extended to encompass a much broader class of methods, making it a unifying principle in numerical analysis. This generality allows practitioners to choose the most appropriate discretization for their specific problem while maintaining the same theoretical guarantees and structure-preserving properties.
+
+[^LampertRanocha2024]:
+    Lampert, Ranocha (2024):
+    Structure-Preserving Numerical Methods for Two Nonlinear Systems of Dispersive Wave Equations
+    [arXiv: 2402.16669](https://arxiv.org/abs/2402.16669)
+
+## 2. Mathematical Foundation
+
+### The Core SBP Property
+
+The fundamental property that defines a first-derivative SBP operator is:
+
+```math
+MD + D^T M = t_R t_R^T - t_L t_L^T
+```
+
+where:
+
+- ``D`` is the derivative operator (matrix)
+- ``M`` is the symmetric, positive definite mass matrix
+- ``t_L = (1,0,...,0)^T`` and ``t_R = (0,...,0,1)^T`` extract boundary values
```

Comment:
```suggestion
- ``\boldsymbol{e}_L = (1,0,...,0)^T`` and ``\boldsymbol{e}_R = (0,...,0,1)^T`` extract boundary values
```
Could we use ``\boldsymbol{e}_L/R`` for these unit vectors to stay consistent with the paper and also make them boldface since they are vectors and below we use boldface to indicate the variable is a vector.